woocommerce/woocommerce
# [REST API] Order/Tax lookup table

### All Submissions:






### Changes proposed in this Pull Request:



Closes #20770  .

### How to test the changes in this Pull Request:

1. Deactivate then reactivate WC to get the latest tables.
2. Create an order with some tax
3. Verify data gets created in table using mySQL
